### PR TITLE
Make tests pass even when julia is started without --color=yes

### PR DESCRIPTION
**Pull request by tkelman**

### CODE REVIEW

File: test/tst_encodeimg.jl [hunk 1]
```diff
@@ -11,7 +11,7 @@ end
 
 @testset "encodeimg 256 small" begin
     @testset "gray square" begin
-        res, h, w = @inferred ImageInTerminal.encodeimg(ImageInTerminal.SmallBlocks(), ImageInTerminal.TermColor256(), gray_square, 2, 2)
+        res, h, w = @inferred ensurecolor(ImageInTerminal.encodeimg, ImageInTerminal.SmallBlocks(), ImageInTerminal.TermColor256(), gray_square, 2, 2)
```
Comment (Evizero) [inline test/tst_encodeimg.jl]: Won't this make `@inferred` test the returntype of `ensurecolor` instead of `encodeimg`

Comment (tkelman) [reply test/tst_encodeimg.jl]: probably. could move the inferred too maybe?

Comment (Evizero) [reply test/tst_encodeimg.jl]: That would be great

**Comment by Evizero** [comment]
~~Could you deactivate `--color=yes` from the travis yml as well while you are at it? If you ran out of time you want to spend on this, then I am ok doing it mysekf after merging this PR~~

**Comment by Evizero** [comment]
Oh, my bad. It seems I activated color on an irrelevant position. Hmm, curious how the tests worked anyway

**Comment by Evizero** [comment]
The error looks unrelated to this PR. I'll look into it. Thanks for doing this

**Pull request merged**